nixos/modules/services/matrix/draupnir.nix

```
@@ -0,0 +1,278 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
+let
+  cfg = config.services.draupnir;
+
+  format = pkgs.formats.yaml {};
+  configFile = format.generate "draupnir.yaml" cfg.settings;
+in
+{
+  #region Options
+  options.services.draupnir = {
+    enable = mkEnableOption (lib.mdDoc "Draupnir, a moderation tool for Matrix");
+
+    accessTokenFile = mkOption {
+      type = with types; nullOr path;
+      default = null;
+      description = lib.mdDoc ''
```

```suggestion
      description = ''
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557
I wont comment on every instance of it.